client/src/inside/components/mar/tests/mar-helper.spec.js

```diff
@@ -0,0 +1,226 @@
+import should from 'should'
+import MarHelper, { MAR_PAGE_KEY, MED_ORDERS_PAGE_KEY } from '../mar-helper'
+import { getMedOrders, getMarRecords } from './mar-test-helper'
+import EhrDefs from '../../../../helpers/ehr-defs-grid'
+import MarEntity from '../mar-entity'
+import MedOrder from '../med-order'
+
+
+let marHelper
+const medOrders = getMedOrders()
+const classMedOrders = getMedOrders(true)
+const marRecords = getMarRecords()
+const classMarRecords = getMarRecords(true)
+
+const getAsLoaded = (key) => {
+  if (key === MED_ORDERS_PAGE_KEY){
+    return Object.assign({}, {
+      pageElements: { table: { tableKey: 'table'}},
+      table: medOrders
+    })
+  } else if (key === MAR_PAGE_KEY) {
+    return Object.assign({}, {
+      pageElements: { table: { tableKey: 'table'}},
+      table: marRecords
+    })
+  }
```

Comment:
can you make this more DRY please.  Perhaps create a list of keys and just check it the incoming key is in the list?

Reply:
I have done so. Was that what you meant? I think it looks cleaner, but let me know if there is any other way I can improve it further. Thank you!